packages/orderbook/test/js_api/gui.test.ts

```
@@ -1508,21 +1520,29 @@ ${dotrainWithoutVaultIds}`;
-			result = await testGui.generateDepositCalldatas();
-			expect(result.error.msg).toBe('Token must be selected: token1');
-			expect(result.error.readableMsg).toBe("The token 'token1' must be selected to proceed.");
-
-			result = await testGui.generateAddOrderCalldata();
-			expect(result.error.msg).toBe('Token must be selected: token1');
-			expect(result.error.readableMsg).toBe("The token 'token1' must be selected to proceed.");
-
-			result = await testGui.generateDepositAndAddOrderCalldatas();
-			expect(result.error.msg).toBe('Token must be selected: token1');
-			expect(result.error.readableMsg).toBe("The token 'token1' must be selected to proceed.");
+			let result1 = await testGui.checkAllowances('0x1234567890abcdef1234567890abcdef12345678');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result1' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)